.github/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,46 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Could we link to Vapor's CoC? That way if for some reason we need to alter it, we don't have to open PRs against all of our repos: https://github.com/vapor/vapor/blob/master/Docs/code_of_conduct.md